tests/object/test.bats

```diff
@@ -0,0 +1,84 @@
+# PURPOSE:
+# I will test the following commands: object create, get, list, update, url, delete, and clean.
+load ../test_setup.bash
+
+# constants
+obj1="test1.txt"
```

Comment:
Are the first two files supposed to be empty?

Reply:
Yes, I just created empty txt files to test the obj commands.